tests/ssp/main.c

```diff
@@ -35,6 +43,9 @@ void test_func(void)
     /* cppcheck-suppress bufferAccessOutOfBounds
      * (reason: deliberately overflowing stack) */
     memset(buffer_to_confuse_compiler, 0, 32);
+
+    /* prevent that memset call is optimized out by LTO */
+    _prevent_memset_lto(buf, 0, 32);
```

Comment:
Is `explicit_bzero()` available with our toolchain?

Reply:
I don't know.